x-pack/plugin/security/src/main/java/org/elasticsearch/xpack/security/authc/ApiKeyService.java

```diff
@@ -1901,13 +1902,13 @@ public void getApiKeys(
             username,
             apiKeyName,
             apiKeyIds,
-            false,
-            false,
+            activeOnly,
```

Comment:
Only real change in API key service -- nice that our code was general enough to support this 🎉